src/app/user-profile/account-settings/account-settings.component.html

```diff
@@ -36,6 +36,19 @@
           (applyChanges)="reloadPage()"
         ></app-privacy-settings>
       </p-tabPanel>
+      <p-tabPanel>
+        <ng-template pTemplate="header">{{ 'PASSWORD.TAB_NAME_PASSWORD' | translate }}</ng-template>
+        <div class="sm:pt-2 pb-2 sm:pb-3 px-3">
+          <ocx-slot
+            *ngIf="isChangePasswordComponentDefined$ | async; else changePasswordPlaceholder"
```

Comment:
I would hide the complete tab if there is no RemoteComponent defined for the slot. Otherwise it looks like an error but maybe the customer does not want to have the change password function in his portal

Reply:
I spoke with Henry about that and we agreed to use a placeholder message.